packages/orchestration/src/examples/auto-stake-it-tap-kit.js

```diff
@@ -57,9 +57,7 @@ const prepareStakingTapKit = (zone, { watch }) => {
         ),
       }),
       transferWatcher: M.interface('TransferWatcher', {
-        onFulfilled: M.call(M.undefined())
```

Comment:
interesting... a `void` return got encoded as "only `undefined`" rather than "I don't care".

Keeping static types consistent with endo patterns is quite a challenge.